coordinator/pom.xml

```diff
@@ -70,6 +70,24 @@
       <scope>test</scope>
       <type>test-jar</type>
     </dependency>
+    <dependency>
+      <groupId>com.tencent.rss</groupId>
+      <artifactId>shuffle-storage</artifactId>
```

Comment:
coordinator shouldn't depend on shuffle-storage module

Reply:
our hdfs related utils are in this module and  we need to support hdfs candidates file.